tutorials/geodetic_to_geocentric.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2019 The Boule Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
+#
+r"""
+.. _geodetic_to_geocentric:
+
+Converting geodetic latitude and height into geocentric latitude and radius
+==============
```

Comment:
```suggestion
Coordinate conversions
==================

```

Reply:
The shorter title fits better on the menu bars. We can always explain it better in the text. There also needs to be blank line after and the `=` need to be the same length as the actual title (I know, that's really annoying)